ui/src/app/components/webcomponents/test-case-steps-list.component.ts

```diff
@@ -191,6 +191,7 @@ export abstract class TestCaseStepsListComponent extends BaseComponent implement
     }
     this.resetAllStepActions();
     testStep.isEditing = false;
+    this.testSteps.content.sort((firstStep,secondStep)=>firstStep.position - secondStep.position);
     this.testSteps.content = [...this.testSteps.content];
```

Comment:
what is need of overwrite? @manikanta-reddy-t

Reply:
technically it is not necessary, i'll check and remove

Reply:
after removing 195 line, when we add a teststep, that step is not being shown until a page refresh has been done

Reply:
plz check the root cause for prop not getting updated for new item push